aproxy.go

```diff
@@ -382,13 +383,19 @@ func HandleConn(conn net.Conn, proxy string) {
 		logger.Info("relay HTTP connection to proxy")
 		RelayHTTP(consigned, proxyConn, logger)
 	default:
-		logger.Error(fmt.Sprintf("unknown destination port: %d", dst.Port))
-		return
+		logger = logger.With("host", "TCP4:"+dst.IP.String()+":"+strconv.Itoa(dst.Port))
```

Comment:
```suggestion
		logger = logger.With("host", fmt.Sprintf("%s:%d", dst.IP.String(), dst.Port))
```
I think maybe we can drop the `TCP4:` prefix to better align with other logging. Also, I think we can use `fmt.Sprintf` to import one fewer package. What do you think?

Reply:
I used your version